pkg/ingress/gateway_test.go

```diff
@@ -17,191 +17,17 @@ import (
 	"github.com/openservicemesh/osm/pkg/announcements"
 	"github.com/openservicemesh/osm/pkg/certificate"
 	tresorFake "github.com/openservicemesh/osm/pkg/certificate/providers/tresor/fake"
-	"github.com/openservicemesh/osm/pkg/configurator"
 	"github.com/openservicemesh/osm/pkg/k8s/events"
 	"github.com/openservicemesh/osm/pkg/messaging"
 )
 
 const (
 	maxTimeToSubscribe           = 500 * time.Millisecond
-	maxTimeForEventToBePublished = 2 * time.Second
+	maxTimeForEventToBePublished = 500 * time.Millisecond
```

Comment:
What's the rationale for these decreases?

Reply:
pretty much everything below is covered in an eventually, and the tests were taking a long time to run for me. 

The originals felt aggressive